web_tool_stress_test.R

```diff
@@ -324,7 +324,7 @@ if (file.exists(file.path(results_path, pf_name, paste0("Equity_results_", calcu
       # Convert capacity (MW)to generation (MWh) for power sector
       equity_annual_profits <- pacta_equity_results %>%
         convert_cap_to_generation(capacity_factors_power = capacity_factors_power) %>%
-        extend_scenario_trajectory(
+        extend_scenario_trajectory_old(
```

Comment:
Similarly to a comment above, reflect on whether or not we need old code.

Reply:
it's still in use